qcodes/instrument/sims/keysight_b1500.yaml

```diff
@@ -64,6 +64,13 @@ devices:
         r: "B1517A,0;B1517A,0;B1520A,0;0,0;0,0;B1530A,0;0,0;0,0;0,0;0,0"
       - q: "TI 1,-16"
         r: "NAI+000.000E-06"
+      - q: "WTDCV 0,0,0,0,0"
+      - q: "WDCV 3,1,0,0,1"
+      - q: "ACT 2,1"
+      - q: "IMP 101"
+      - q: "LMN 0"
+      - q: "MT 0,0.1,7"
+      - q: "FMT 1,0"
```

Comment:
How did it work before without these?

Reply:
since these are called with `write`, nothing was "returned", so from `.write(` call point of view everything was fine. However, since these commands were not known to the yaml file, they generated "command error" errors in the error queue of the visa resource (which mimics what the instrument does as well), so if you were to check the error queue after you "write", then you would see the "problem". and the latter we plan to do as a later improvememnt.